seqr/views/utils/permissions_utils.py

```diff
@@ -19,14 +20,50 @@ def get_project_and_check_permissions(project_guid, user, **kwargs):
     return project
 
 
-def has_project_permissions(project, user, can_edit=False, is_owner=False):
+def is_staff(user, session):
+    """
+    Background of this function.
+
+    The staff management with an AnVIL group is problematic because it hard to tracking the changes in the group.
+    Since we keep user models on seqr, the 'is_staff' is available from the model.
+    """
+    return user.is_staff
+
+
+def has_perm(user, permission_level, project, session):
+    # the 'service_account_session' below will be replaced by 'session' after seqr client ID is whitelisted
+    is_staff_user = is_staff(user, session)
+    if is_staff_user and not project.disable_staff_access:
+        return True
+    session = service_account_session
```

Comment:
please do not redefine a variable inside a function, it makes it impossible to read. `session` is passed in so it can't also be this imported constant

Reply:
Got it.